user-account-management.md

```diff
@@ -107,14 +196,18 @@ TiDB stores passwords in the `mysql.user` system database. Operations that assig
     skip-grant-table = true
     ```
 
-2. Use `root` to log in and then modify the password:
+2. Start TiDB with the modified configuration. Use `root` to log in and then modify the password:
 
     ```bash
     mysql -h 127.0.0.1 -P 4000 -u root
     ```
 
+When the `skip-grant-table` is set, starting the TiDB process will check the user to be an administrator of the operating system, and only the `root` user of the operating system can start the TiDB process.
+
 ## `FLUSH PRIVILEGES`
 
+Information related to users and privileges is stored in the TiKV server, and TiDB caches this information inside the process. Generally, modification of the related information through CREATE USER, GRANT and other statements takes effect quickly within the entire cluster. If the operation is affected by some factors such as network temporary unavailable, it will take effect in about 15 minutes as TiDB will periodically reload the cache information.
```

Comment:
```suggestion
Information related to users and privileges is stored in the TiKV server, and TiDB caches this information inside the process. Generally, modification of the related information through `CREATE USER`, `GRANT`, and other statements takes effect quickly within the entire cluster. If the operation is affected by some factors such as temporarily unavailable network, the modification will take effect in about 15 minutes because TiDB will periodically reload the cache information.
```